### ISSUE TITLE
Replace transitiveSfOverlap with geo:sfOverlap in the mb<>cc linkset pre-conditioning scripts

### ISSUE THREAD

**Issue by benjaminleighton** [CSIRO-enviro-informatics/mb16cc-linkset#1]
http://linked.data.gov.au/def/geox#transitiveSfOverlap  describes things that overlap by virtue of a relationship a contains b  b within c, however in the case of meshblocks to catchment linkset the b object that indicates a and b transitive overlap is a synthetic intersection object. The intent of trasntiveSfOverlap seems to be to make an assertion about the provenance of an overlap to indicate that an overlap has been determine between two objects because a third object is present that is fully within both objects. In the mb<>cc linkset cache the "third" linking object only exists by virtue of a synthetic intersection object and not a first class entity in its own right. If we consider these synthetic intersection objects to sufficient to describe a transitiveSfOverlap this will pollute / dilute cases where transitiveSfOverlap is meant to identify a meaningful derived relationship from first class objects.

**Comment by dr-shorthair** [comment]
related to https://github.com/CSIRO-enviro-informatics/geosparql-ext-ont/issues/1

**Comment by dr-shorthair** [comment]
I agree that there is no need for `goex:transitiveSfOverlap` in the data (or the ontology). 
These are just `geo:sfOverlap` which have been computed using a particular path (i.e. with a specific provenance).
